tests/runners/airflow/demo/model_store.py

```diff
@@ -0,0 +1,45 @@
+import pickle
+import time
+
+import boto3
+
+BUCKET = 'ni-ml-ab-dev'
```

Comment:
how will this example work for users? in terms of requiring S3 buckets to exist etc.?

Reply:
Wouldn't work for them, they would need an s3 bucket